dev/MRTCore/README.md

```diff
@@ -0,0 +1,67 @@
+# Welcome to the MRT Core Repo!
+
+## Overview 
+
+### What is MRT Core?
+Thanks for asking! MRT Core was born from the need for [Project Reunion](https://aka.ms/projectreunion) applications to use MRT - Modern Resource Technology. MRT was introduced to Windows in the Windows 8 timeframe and has evolved as the resource management system since that time. You can use these links to learn more about [MRT and resource management](https://docs.microsoft.com/en-us/windows/uwp/app-resources/resource-management-system) as well as the benefits of [globalization and localization](https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/globalizing-portal).
+
+### So what does it do?
+At its *core* 😎 MRT Core provides support to build resource Package Resource Index (PRI) files, and load resources from PRI files. [Learn more about PRI files at this link.](https://docs.microsoft.com/en-us/windows/uwp/app-resources/resource-management-system)
+
+## Getting Started
+Install the NuGet package from [https://aka.ms/MRTCore](https://aka.ms/MRTCore), and you are ready to build PRI files from Visual Studio
+and use MRT Core APIs to load resources in your app.
+
+The C APIs can be found in [mrm.h](mrt/Core/src/MRM.h). The WinRT APIs can be found in [Microsoft.ApplicationModel.Resources.idl](mrt/Microsoft.Applicationmodel.Resources/src/Microsoft.Applicationmodel.Resources.idl).
+
+## Build and Test
+Load [MrtCore.sln](mrt/MrtCore.sln) in Visual Studion 2019, and build the solution. 
```

Comment:
Did you add the SLN here (or the projects within) to the overall Reunion SLN?

Reply:
we want to keep the building unit for MRTCore so that we don't have to build the world.

---
In reply to: [488400606](https://github.com/microsoft/ProjectReunion/pull/185#discussion_r488400606) [](ancestors = 488400606)